backend/pkg/api/handle_topic_messages_integration_test.go

```diff
@@ -1036,6 +1038,8 @@ func (s *APIIntegrationTestSuite) TestPublishMessages() {
 		assert.Equal("543", obj2.Id)
 		assert.Equal("item_sr_0", obj2.Name)
 		assert.Equal(timestamppb.New(objTime), obj2.CreatedAt)
+
+		protocmp.AssertProtoEqual(t, originalObject, &obj2)
```

Comment:
protobuf serialization is not deterministic ([docs](https://protobuf.dev/programming-guides/serialization-not-canonical/)), and the above original assertion `assert.Equal(expectedData, record.Value)` is not technically correct.
This changes the test to more proper proto comparison and assertion.